src/main/java/net/fabricmc/meta/utils/MinecraftLauncherMeta.java

```diff
@@ -80,6 +90,25 @@ public String getTime() {
 		public String getReleaseTime() {
 			return releaseTime;
 		}
+
+		public String getMajorVersion() {
+			String json = null;
+			try {
+				json = IOUtils.toString(new URL(url), StandardCharsets.UTF_8);
+			} catch (IOException e) {
+				e.printStackTrace();
+			}
+			return GSON.fromJson(json, Meta.class).getAssets();
+		}
+	}
+
+	public static class Meta {
+
+		String assets;
```

Comment:
Im not sure the asset id always matches the target.

Reply:
I haven't seen a case where it hasn't.
Here's the output it generates: https://gist.github.com/dexman545/4ad16ef4e1de9d10f9042b54fc5fd8ab